precompile/fee_config_manager.sol

```diff
@@ -0,0 +1,29 @@
+// (c) 2022-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.8.0;
+
+interface FeeConfigManagerInterface {
+    // Set [addr] to have the admin role over the minter list
+    function setAdmin(address addr) external;
+
+    // Set [addr] to be enabled on the minter list
+    function setEnabled(address addr) external;
+
+    // Set [addr] to have no role over the minter list
+    function setNone(address addr) external;
```

Comment:
can we update these comments to reflect that these roles are for the fee config manager?